desc/objectives/_neoclassical.py

```diff
@@ -454,7 +454,7 @@ def compute(self, params, constants=None):
             self._keys,
             params,
             get_transforms(self._keys, eq, grid, jitable=True),
-            get_profiles(self._keys, eq, grid, jitable=True),
+            get_profiles(self._keys, eq, grid),
```

Comment:
Would it be better to get profiles inside build? Or maybe these are changing throughout optimization. same thing on ripple branch

Reply:
I am pretty sure we can get them inside of build? if we are on the same flux surface, profiles are only functions of rho (for the most part). so that might at leats remove one possible source of error